packages/core/upload/admin/custom.d.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference types="vite/client" />
+
+import { type StrapiTheme } from '@strapi/design-system';
+
+import type { Modules } from '@strapi/types';
+
+declare module 'styled-components' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  export interface DefaultTheme extends StrapiTheme {}
+}
+
+interface BrowserStrapi {
+  backendURL: string;
```

Comment:
What if you only provided this backendURL line since it's all we need?

Reply:
done